src/components/SelectedFolderRender.js

```diff
@@ -0,0 +1,45 @@
+import { getTimeAgo } from '../getTimeAgo';
+import FloatingAddBtn from './FloatingAddBtn';
+import './FloatingAddBtn.css';
+
+function formatDate(value) {
+  const date = new Date(value);
+  return `${date.getFullYear()}. ${date.getMonth() + 1}. ${date.getDate()}`;
+}
+
+function SelectedFolderRender({ selectedFolder }) {

```

Comment:
- Render는 없어도될거에요. 리액트컴포넌트는 하나의 UI를 표현하는 함수라서, 일반적인 함수네이밍과 다르게 하시는게 좋습니다.
- 단수형과 복수형은 큰 차이가 있습니다.  

```suggestion
function SelectedFolderLinks({ links }) {
```